code/__HELPERS/maths.dm

```diff
@@ -80,6 +80,41 @@
 			line += locate(current_x_step, current_y_step, starting_z)
 	return line
 
+/**
+ * Get a list of turfs in a perimeter given the `center_atom` and `radius`.
+ * Automatically rounds down decimals and does not accept negative radius as they dont play well with it.
+ * Is efficient on large circles but ugly on small ones
+ * Uses [Jesko`s method to the midpoint circle Algorithm](https://en.wikipedia.org/wiki/Midpoint_circle_algorithm).
+ */
+/proc/get_perimeter(atom/center, radius)
+	if(radius < 0)
```

Comment:
This will not early return if radius is 0, is that normal?

Reply:
youre right i should exclude 0